src/iconsManifest/supportedExtensions.ts

```diff
@@ -253,6 +253,24 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'ng_routing_dart',
+      extensions: ['routes.dart'],
```

Comment:
Is there a reason for this duplication of entries? If I understood this right, just adding `routes.dart`, `routes.js`, etc, to the existing entries would be enough.

Thanks for your contribution!

Reply:
Thanks for pointing that out! I think I got confused since there were multiple entries that used e.g. `ng_routing_ts` as icon.  I didn't think of the `filename` property being the reason they might have been kept separate.

```ts
    {
      icon: 'ng_routing_ts',
      extensions: ['routes.ts'],
      format: FileFormat.svg,
      disabled: true,
    },
    // ...
    {
      icon: 'ng_routing_ts',
      extensions: ['app-routing.module.ts'],
      filename: true,
      format: FileFormat.svg,
      disabled: true,
    },
```

I've updated the PR with your suggestion, to just add `routes.(ts|js|dart)` to existing `extensions` arrays for existing entries. Let me know if I missed anything else 🙂